CMake/DisplayCluster.cmake

```diff
@@ -1,14 +1,4 @@
 
-set(DISPLAYCLUSTER_PACKAGE_VERSION 0.4)
-set(DISPLAYCLUSTER_REPO_URL https://github.com/BlueBrain/DisplayCluster.git)
-
-set(DISPLAYCLUSTER_DEPENDS
-  REQUIRED Boost Deflect FCGI FFMPEG MPI OpenGL Qt5Concurrent Qt5Core
-           Qt5Declarative Qt5Network Qt5OpenGL Qt5Svg Qt5WebKitWidgets
-           Qt5Widgets Qt5Xml Qt5XmlPatterns
-  OPTIONAL bluebrain Poppler TUIO OpenMP)
-set(DISPLAYCLUSTER_BOOST_COMPONENTS "program_options date_time serialization unit_test_framework regex system thread")
-set(DISPLAYCLUSTER_POPPLER_COMPONENTS "Qt5")
 set(DISPLAYCLUSTER_DEB_DEPENDS libavutil-dev libavformat-dev libavcodec-dev
```

Comment:
You could even move the remains of this file to the top-level CMake or CPackConfig, if you like.